### ISSUE TITLE
Documentation file is broken

### ISSUE THREAD

**Issue by GoogleCodeExporter** [msndevs/msnp-sharp#191]
```
What steps will reproduce the problem?
1. Download latest documentation file from
http://code.google.com/p/msnp-sharp/source/browse/#svn/docs/v30 
2. Open the file

What is the expected output? What do you see instead?
I get "Navigation to the webpage was cancelled" when opening the file, and
"This program cannot display the webpage" errors when clicking on any items
in the Contents/Index/Search list.

What version of the product are you using? (MSNPSharp, OS, Mono etc.)
Tested on Windows 7 (IE8) and Windows Server 2003 (IE8)

Please provide any additional information below:
I suspect IE has something to do with it, as you see an IE error page in
the help viewer. 
```

Original issue reported on code.google.com by `zangerbo...@gmail.com` on 24 Apr 2010 at 4:22

**Comment by GoogleCodeExporter** [comment]
```
I am afraid that you need to check the settings of your IE or windows, I test 
that 
file on several systems and my Windows, everything is ok.
```

Original comment by `freezing...@gmail.com` on 25 Apr 2010 at 5:41
- Changed state: **Invalid**

**Comment by GoogleCodeExporter** [comment]
```
BTW, if you are using Chinese system, please place the file into a directory 
without 
Chinese, or go to Control Panel and set your region into China.
```

Original comment by `freezing...@gmail.com` on 25 Apr 2010 at 5:46

**Comment by GoogleCodeExporter** [comment]
```
Or right-click the chm file, then choose "unlock the file".
```

Original comment by `freezing...@gmail.com` on 25 Apr 2010 at 5:50

**Comment by GoogleCodeExporter** [comment]
Original comment by `freezing...@gmail.com` on 25 Apr 2010 at 5:52
- Changed state: **WontFix**

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the quick replies. I was almost certain it wasn't me because i 
tested it
on several (English) systems, but the "unblock file" button in properties of 
the file
worked. Thanks!
```

Original comment by `zangerbo...@gmail.com` on 26 Apr 2010 at 12:43
